docs/components/select/use-cases.md

```diff
@@ -2,54 +2,75 @@
 
 ```js script
 import { html } from '@mdjs/mdjs-preview';
+import { Required } from '@lion/ui/form-core.js';
+import { loadDefaultFeedbackMessages } from '@lion/ui/validate-messages.js';
 import '@lion/ui/define/lion-select.js';
+loadDefaultFeedbackMessages();
 ```
 
 ## Pre-select
 
 You can preselect an option by setting the property modelValue.
 
-```js preview-story
-export const preSelect = () => html`
-  <lion-select name="favoriteColor" label="Favorite color" .modelValue=${'hotpink'}>
-    <select slot="input">
-      <option selected hidden value>Please select</option>
-      <option value="red">Red</option>
-      <option value="hotpink">Hotpink</option>
-      <option value="teal">Teal</option>
-    </select>
-  </lion-select>
-`;
+```html preview-story
+<lion-select name="favoriteColor" label="Favorite color" .modelValue=${'hotpink'}>
+  <select slot="input">
+    <option selected hidden value>Please select</option>
+    <option value="red">Red</option>
+    <option value="hotpink">Hotpink</option>
+    <option value="teal">Teal</option>
+  </select>
+</lion-select>
 ```
 
 ## Disabled
 
 You can disable an option by adding the `disabled` attribute to an option.
 
-```js preview-story
-export const disabledOption = () => html`
-  <lion-select name="favoriteColor" label="Favorite color">
-    <select slot="input">
-      <option selected hidden value>Please select</option>
-      <option value="red">Red</option>
-      <option value="hotpink" disabled>Hotpink</option>
-      <option value="teal">Teal</option>
-    </select>
-  </lion-select>
-`;
+```html preview-story
+<lion-select name="favoriteColor" label="Favorite color">
+  <select slot="input">
+    <option selected hidden value>Please select</option>
+    <option value="red">Red</option>
+    <option value="hotpink" disabled>Hotpink</option>
+    <option value="teal">Teal</option>
+  </select>
+</lion-select>
 ```
 
 Or by setting the `disabled` attribute on the entire `lion-select` field.
 
+```html preview-story
+<lion-select name="favoriteColor" label="Favorite color" disabled>
+  <select slot="input">
+    <option selected hidden value>Please select</option>
+    <option value="red">Red</option>
+    <option value="hotpink">Hotpink</option>
+    <option value="teal">Teal</option>
+  </select>
+</lion-select>
+```
+
+## Validation
```

Comment:
Maybe we should mention here that it inherits from LionField/FormControl and validation works the same for all FormControls.

For Combobox the example made sense since it has a specific Validator

Reply:
So no visual code block, and just a link to `./fundatementals/systems/form/validate.md` page.